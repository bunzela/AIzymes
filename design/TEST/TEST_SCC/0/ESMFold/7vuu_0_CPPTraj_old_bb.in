parm /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/0/7vuu_RosettaDesign_0.pdb
trajin  /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/0/7vuu_RosettaDesign_0.pdb
strip   :5TS
strip   !@C,N,O,CA
trajout /mnt/vast-standard/home/hansadrian.bunzel01/u14852/AIzymes/design/TEST/TEST_SCC/0/ESMFold/7vuu_0_old_bb.pdb
