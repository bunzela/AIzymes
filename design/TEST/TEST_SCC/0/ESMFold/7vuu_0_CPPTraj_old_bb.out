cpptraj: error while loading shared libraries: libfftw3.so.3: cannot open shared object file: No such file or directory
